opm/autodiff/FlowMain.hpp

```diff
@@ -423,8 +423,16 @@ namespace Opm
             OpmLog::addBackend( "STREAMLOG", streamLog);
             std::shared_ptr<StreamLog> debugLog = std::make_shared<EclipsePRTLog>(debugFile, Log::DefaultMessageTypes, false, output_cout_);
             OpmLog::addBackend( "DEBUGLOG" ,  debugLog);
+            const auto& msgLimits = eclipse_state_->getSchedule()->getMessageLimits();
+            const std::map<int64_t, int>limits = {{Log::MessageType::Note, msgLimits.getMessagePrintLimit(0)},
```

Comment:
Nitpick: there should be a space between the type and the variable. I am a bit surprised this compiled, but I guess the template angle brackets separate the tokens.